dirty_cat/test/test_fuzzy_join.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+import pytest
+from dirty_cat import FuzzyJoin
+
+
+@pytest.mark.parametrize(
+    "analyzer, return_distance", [("char", True), ("char_wb", True), ("word", True)]
+)
+def test_fuzzy_join(analyzer, return_distance):
```

Comment:
maybe we should add a small performance test? (like on this example, check that it matches most team's names)